llvm/cmake/modules/TableGen.cmake

```diff
@@ -190,9 +190,13 @@ macro(add_tablegen target project)
   endif()
 
   # FIXME: Quick fix to reflect LLVM_TABLEGEN to llvm-min-tblgen
+  set(RAW_LLVM_TABLEGEN ${LLVM_TABLEGEN})
+  if(NOT "${RAW_LLVM_TABLEGEN}" STREQUAL "")
+    get_filename_component(RAW_LLVM_TABLEGEN ${RAW_LLVM_TABLEGEN} NAME_WE)
```

Comment:
Would it match if `${LLVM_TABLEGEN}` points to `X:/out-of-tree/path/to/llvm-tblgen.exe`? I don't think it'd be desirable.